### ISSUE TITLE
[Blog] Blog articles not populating 

### ISSUE THREAD

**Issue by j-zerah** [status-im/status-network-site#7]
Blog articles from the various Status Network blogs (Status, Nimbus, Embark) are not populating on the homepage nor blog itself

https://news.statusnetwork.com/#

<img width="1250" alt="Screen Shot 2020-01-27 at 4 45 40 PM" src="https://user-images.githubusercontent.com/33129508/73216712-7ec9bd80-4124-11ea-879c-7e3b2c4c8e00.png">

@NistorCristian

**Comment by NistorCristian** [comment]
@jakubgs We have this error here:

Access to XMLHttpRequest at 'https://embark.status.im/atom.xml' from origin 'https://news.statusnetwork.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.

Can we add an extra response header on embark so we can fetch the blog posts from there. This is what triggers the rest. I'll update in the next days the theme so I can catch the error if a blog fails. Until then can you help us with this, please?

**Comment by jakubgs** [comment]
Will check it out in the morning.

**Comment by jakubgs** [comment]
Though it might be that it's just the domain change that caused this. If we use https://blog.embarklabs.io/atom.xml directly does it still fail?

**Comment by jakubgs** [comment]
I'm pretty sure it's the domain change. I'll test a fix:
```
 $ curl -sv 'https://blog.embarklabs.io/atom.xml' \
      -H 'Origin: news.statusnetwork.com' \
      > /dev/null 2>&1 | grep access-control
< access-control-allow-origin: *
```

**Comment by jakubgs** [comment]
I've pushed the ba4d70b18a2270a693c3f97bc0603e9f51b8122d fix to `develop`, seems to work:
![embark_atmo_xml_fetch](https://user-images.githubusercontent.com/2212681/73254582-fba97580-41be-11ea-949c-dd7b97d7ac69.png)
But we probably need a better fix to not let it fall on its face when one source fails.

**Comment by j-zerah** [comment]
@jakubgs @NistorCristian what about all of the other blogs:
- our.status.im
- news.nimbus.team
- https://vac.dev/research-log/

and the actual articles published to https://news.statusnetwork.com/#

Nothing seems to be populating

**Comment by NistorCristian** [comment]
Fixed. Try to clear cache or incognito.

**Comment by jakubgs** [comment]
Looks correct to me:
![news_status_netowrk_embark](https://user-images.githubusercontent.com/2212681/73276386-e5b0aa80-41e8-11ea-996c-a45d43c8afc9.png)
@NistorCristian did you also modify the News site JS to not fail entirely if one feed fails?

**Comment by jakubgs** [comment]
I think this is done @j-zerah, reopen it if you still see the issues. Thanks Cristian!

**Comment by j-zerah** [comment]
Latest blog content is still not populating on the homepage. It is only showing old content. And even looks to only be showing Vac content

https://statusnetwork.com/

<img width="1276" alt="Screen Shot 2020-01-29 at 11 54 31 AM" src="https://user-images.githubusercontent.com/33129508/73378171-2e6e6f00-428e-11ea-987d-ed776d8d1de5.png">


@jakubgs @NistorCristian

**Comment by jakubgs** [comment]
Oh, right, I didn't push the fix to prod on Status Network, just to `dev`. Silly me.

But I found another issue with https://news.nimbus.team/:
![nimbus_news_ghost_issue](https://user-images.githubusercontent.com/2212681/73379634-e87fce00-42c2-11ea-86a2-3541f3eec8ee.png)

**Comment by jakubgs** [comment]
Oh, seems like it was temporary. Now it's fine:
![sn_news](https://user-images.githubusercontent.com/2212681/73393905-c98e3580-42dc-11ea-9dac-f93842588eae.png)
